### PR TITLE
finished ui on apio1

### PR DESCRIPTION
**Pull request by Piro2maniC**
Finished up my UI in API01.jsx component please have a look

### CODE REVIEW

File: src/pages/HomePage/components/HomePageSectionOne/components/API01.jsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ const CarValueController = () => {
 
         {numericValue !== null && (
           <p className={styles.result}>
-            The value of your car is:{" "}
+            The value if your car is:{" "}
```
Comment (rhyzJ) [inline src/pages/HomePage/components/HomePageSectionOne/components/API01.jsx:84]: change if to of

Comment (christophermissionready) [inline src/pages/HomePage/components/HomePageSectionOne/components/API01.jsx:84]: You have made a spelling mistake here. Change "if" to "of".

**Review by christophermissionready — COMMENTED** [review]
Please change spelling error. Other than that looks v good

**Review by rhyzJ — APPROVED** [review]
looks fab

**Pull request merged**